server/sonar-server/src/main/java/org/sonar/server/component/db/ComponentDao.java

```diff
@@ -172,4 +177,22 @@ protected void doDeleteByKey(DbSession session, String key) {
     return mapper(session).selectProjectsFromView("%." + viewUuid + ".%", projectViewUuid);
   }
 
+  public List<ComponentDto> selectProvisionedProjects(DbSession session, SearchOptions searchOptions, @Nullable String query) {
+    Map<String, String> parameters = new HashMap<>();
+    parameters.put("qualifier", Qualifiers.PROJECT);
+    if (query != null) {
+      parameters.put("query", "%" + query + "%");
+    }
+    return session.selectList("org.sonar.core.component.db.ComponentMapper.selectProvisionedProjects", parameters,
```

Comment:
is there any way to avoid having such constants in code? 
Can't you use  `ComponentMapper#selectProvisionedProjects`?